assets/js/modules/sign-in-with-google/components/settings/SettingsForm.js

```diff
@@ -62,6 +89,58 @@ export default function SettingsForm() {
 					<Cell size={ 12 }>
 						<OneTapToggle />
 					</Cell>
+					<Cell size={ 12 }>
+						<SettingsGroup
+							title={ __(
+								'Users can create new accounts',
+								'google-site-kit'
+							) }
+						>
+							<p>
+								{ anyoneCanRegister
+									? sprintf(
+											/* translators: %s: Sign in with Google service name */
+											__(
+												'Users can create new accounts on this site using %s. ',
+												'google-site-kit'
+											),
+											_x(
+												'Sign in with Google',
+												'Service name',
+												'google-site-kit'
+											)
+									  )
+									: sprintf(
+											/* translators: %s: Sign in with Google service name */
+											__(
+												'Only existing users can use %s to access their accounts. ',
+												'google-site-kit'
+											),
+											_x(
+												'Sign in with Google',
+												'Service name',
+												'google-site-kit'
+											)
+									  ) }
+								{ hasCapability &&
+									! isMultisite &&
+									createInterpolateElement(
+										__(
+											'You can change this setting in <a>Settings->General->Membership.</a>',
```

Comment:
Let's use the style WordPress uses in its documentation when describing the location of things in wp-admin. See https://wordpress.org/documentation/article/administer-your-blog/

```suggestion
											'You can change this setting in <a>Settings > General > Membership</a>.',
```

Also, I wonder if we can use placeholders to use core's translations for these page names (i.e. `__( 'Settings' )` - no text domain = core)? I'm not sure we can since these are in JS but it's worth looking into.

Reply:
> Also, I wonder if we can use placeholders to use core's translations for these page names (i.e. __( 'Settings' ) - no text domain = core)? I'm not sure we can since these are in JS but it's worth looking into

Doesn't look like core translation can be loaded, it only support plugin text domain. From official documentation:
> Make sure to also specify the Text Domain for your translations in the header of your plugin file. Otherwise the translations will not be picked up by [translate.wordpress.org](https://translate.wordpress.org/).

Couldn't find other reference to include core translation, besides the usual localisation practice of passing the strings to the JS as global variables, etc.

Style has been updated to match how WP uses it in documentation